images/cleaner/cleaner.sh

```diff
@@ -1,5 +1,78 @@
 #!/bin/sh
 
+set -e
+
 echo "Running Connectivity Proxy Cleanup script"
-kubectl get pods -n kyma-system
-exit 0
\ No newline at end of file
+
+# check if statefulset connectivity-proxy does not exist and if so, exit  with 0
+if ! kubectl get statefulset -n kyma-system connectivity-proxy &> /dev/null; then
+  echo "Connectivity Proxy does not exist, exiting"
+  exit 0
+fi
+
+echo "Connectivity Proxy is installed"
+
+echo "Annotate all existing Connectivity Proxy Service Mappings"
+
+mappings=$(kubectl get servicemappings.connectivityproxy.sap.com -o jsonpath='{range .items[*]}{.metadata.name}{"\n"}{end}')
+
+for mapping in $mappings; do
+
+  echo "Applying annotations to service mapping $mapping"
+
+  kubectl annotate servicemappings.connectivityproxy.sap.com "$mapping" \
+    io.javaoperatorsdk/primary-name=connectivity-proxy \
+    io.javaoperatorsdk/primary-namespace=kyma-system \
+
+done
+
+echo "Removing all Connectivity Proxy workloads"
+
+kubectl delete statefulset -n kyma-system connectivity-proxy --ignore-not-found
+kubectl delete deployment -n kyma-system connectivity-proxy-restart-watcher --ignore-not-found
+kubectl delete deployment -n kyma-system connectivity-proxy-sm-operator --ignore-not-found
+
+echo "Removing all Connectivity Proxy services"
+
+kubectl delete service -n kyma-system connectivity-proxy --ignore-not-found
+kubectl delete service -n kyma-system connectivity-proxy-smv --ignore-not-found
+kubectl delete service -n kyma-system connectivity-proxy-tunnel --ignore-not-found
+kubectl delete service -n kyma-system connectivity-proxy-tunnel-0 --ignore-not-found
+kubectl delete service -n kyma-system connectivity-proxy-tunnel-healthcheck --ignore-not-found
```

Comment:
`connectivity-proxy-tunnel` is deleted twice. Its not a problem actually, but I will remove it either way